src/Player.cpp

```diff
@@ -43,12 +43,19 @@ void Player::setScore() {
 	while (backLoop > name_.size()) {
 		if (isdigit(data_.at(backLoop)))
 			points_ += seti(data_, backLoop);
+        else if(data_.at(backLoop)=='/') points_ += (10 - seti(data_, backLoop-1));

```

Comment:
Readability of this code is poor. You can add braces to every case or maybe extract those specific cases info separate functions with proper names?

Reply:
It's changed now